run

```diff
@@ -18,18 +18,24 @@ MANIFEST_FILE=$FLYWHEEL_BASE/manifest.json
 ##############################################################################
 # Parse configuration
 
-if [[ -f $CONFIG_FILE ]]; then
-  eval $(jq -r '.config | to_entries[] | "config_\(.key)=\(.value)"' $CONFIG_FILE)
-else
-  CONFIG_FILE=$MANIFEST_FILE
-  eval $(jq -r '.config | to_entries[] | "config_\(.key)=\(.value.default)"' $CONFIG_FILE)
-fi
+function parse_config {
+
+  CONFIG_FILE=$FLYWHEEL_BASE/config.json
+  MANIFEST_FILE=$FLYWHEEL_BASE/manifest.json
+
+  if [[ -f $CONFIG_FILE ]]; then
+    echo -e "$(cat $CONFIG_FILE | jq -r '.config.'$1)"
```

Comment:
Is the `-e` there to allow whitespace? 
It does allow for some malicious behavior (e.g. the value `/*/*/*/*`)

Reply:
That was the idea, but does not apply here. Removed.